externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/FlinkSQLEngine.scala

```diff
@@ -72,6 +75,20 @@ object FlinkSQLEngine extends Logging {
     try {
       val flinkConfDir = CliFrontend.getConfigurationDirectoryFromEnv
       val flinkConf = GlobalConfiguration.loadConfiguration(flinkConfDir)
+
+      val executionTarget = flinkConf.getString(DeploymentOptions.TARGET)
+      // set cluster name for per-job and application mode
+      executionTarget match {
+        case "yarn-per-job" | "yarn-application" =>
+          val appName = s"kyuubi_${user}_flink_${Instant.now}"
+          flinkConf.setString("yarn.application.name", appName)
+        case "kubernetes-application" =>
+          val appName = s"kyuubi-${user}-flink-${Instant.now}"
+          flinkConf.setString("kubernetes.cluster-id", appName)
+        case null =>
```

Comment:
Currently, we only support flink local(standalone) mode. Can we only print the target for now?

Reply:
Yes, we don't need to set app names for standalone clusters. 

Do you mean for now we should only validate and print the value of the execution target to ensure we are in standalone mode? But previously I see discussions saying that we are aiming for the application mode. Am I missing something?

Reply:
> Do you mean for now we should only validate and print the value of the execution target to ensure we are in standalone mode? 

Yes

> But previously I see discussions saying that we are aiming for the application mode. 
also yes

> Am I missing something?
No, nothing you have missed. I just suggest we say that it's an error after we would support application mode？ WDYT？

Reply:
After rethink about this. It's also OK for me about the current behavior.

Reply:
After rethink about this. It's also OK for me about the current behavior.